tests/foreman/sys/test_fam.py

```diff
@@ -231,9 +231,18 @@ def test_positive_run_modules_and_roles(module_target_sat, setup_fam, ansible_mo
     if ansible_module in FAM_TEST_LIBVIRT_PLAYBOOKS:
         module_target_sat.configure_libvirt_cr()
 
+    env = [
+        'NO_COLOR=1',
+        'PYTEST_DISABLE_PLUGIN_AUTOLOAD=1',
+        'ANSIBLE_HOST_PATTERN_MISMATCH=ignore',
+    ]
+
+    if settings.server.is_ipv6 and ansible_module in ['redhat_manifest']:
+        env.append(f'HTTPS_PROXY={settings.http_proxy.http_proxy_ipv6_url}')
```

Comment:
I'd recommend using enable_ipv6_system_proxy() helper here instead of this

Reply:
That would enable the proxy for all following test, I don't want to do that.

Reply:
@evgeni No, I mean just enable system proxy when `ansible_module in ['redhat_manifest']`, so it would behave same as you implemented, right?

Reply:
That function writes to bashrc, so when the next test runs, it runs on the same system and the proxy is still enabled.